### PR TITLE
feature/FOUR-10083: Submit Actions in Pane Navigation

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
As a Participant, I want to have the ability to perform the actions that are available when a Task is open in the preview pane. (the same thata will happen when I open teh task in the normal way). This ensures that the completion of the Task will be done inside the pane.

For example if there is a Form that has to be submited, this “Submit Button” should work and give the mesagge of correct or wrong ejecution.

## Solution
- https://processmaker.atlassian.net/browse/FOUR-10063

## How to Test
1. Create a request in the TASK
2. Open the preview
3. Check the screen related and make changes
4. Click on the Submit

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-10083

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Environment Fall
ci:next

### CODE REVIEW

File: resources/js/tasks/components/TasksPreview.vue [hunk 1]
```diff
@@ -159,6 +159,22 @@ export default {
     },
     onClose() {
       this.showPreview = false;
+      this.linkTasks1 = "";
```
Comment (fagubla) [inline resources/js/tasks/components/TasksPreview.vue:162]: Create a new Method to reset the default values

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8e93a7f3b3.eng.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8e93a7f3b3.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-8e93a7f3b3.eng.processmaker.net

**Pull request merged**